sqlx-core/src/postgres/options/mod.rs

```diff
@@ -345,3 +356,244 @@ fn default_host(port: u16) -> String {
     // fallback to localhost if no socket was found
     "localhost".to_owned()
 }
+
+/// try to load a password from the various pgpass file locations
```

Comment:
This file is getting big, *maybe* these helper functions should live in their own sub-module.